src/hint_cli.py

```diff
@@ -56,9 +60,51 @@ def parse(argv=None):
     return path, config, action, args
 
 
+def path_last_deploy(path):
+    return path + "/.last_deploy"
+
+
+def save_config(path, config, cfg):
+    dat = {"config": config,
+           "time": time.time(),
+           "data": cfg}
+    with open(path_last_deploy(path), "wb") as f:
+        pickle.dump(dat, f)
+
+
+def read_config(path):
+    with open(path_last_deploy(path), "rb") as f:
+        dat = pickle.load(f)
+    return dat
+
+
+def load_config(path, config=None, refresh=True):
+    if os.path.exists(path_last_deploy(path)):
+        dat = read_config(path)
+        if refresh:
+            print("[Reloaded configuration '{}' ({} s old)]".format(
```

Comment:
Might be kind to string format this to d, h, m, s

Reply:
I've pulled in a small dep that does a nice job of this